README.md

```diff
@@ -48,6 +48,56 @@ made available on the GitHub releases page: https://github.com/MilesCranmer/rip2
 
 To install, simply open the archive and move the binary somewhere you can run it.
 
+### Nix
+
+This repository is flake-compatible, and backwards-compatible with non-flake systems using `flake-compat`. It uses `naersk` to build the Rust package from source.
+
+#### Try Out Before Installing (With Flakes)
```

Comment:
Could we wrap the detailed instructions with a toggle?

```
<details>
<summary>
…
</summary>

(Full installation instructions)

</details>
```

Then it won’t take up as much room in the readme (see cargo instructions for comparison)

Reply:
See https://gist.github.com/Ashton-W/864d09d71f7cb8e5b47ebaac02b98455

Reply:
Added that in the recent commit